### ISSUE TITLE
English campaign signup should send English transactional

### ISSUE THREAD

**Issue by deezone** [DoSomethingArchive/legacy-website#5563]
Brazilian user signing up for English campaign should get transactional in English, currently `mb-campaign-signup-BR` is sent rather than `mb-campaign-signup-EN`. See example payload below:

```
(
    [activity] => campaign_signup
    [email] => dlee+THORTest101@dosomething.org
    [uid] => 3118772
    [user_language] => pt-br
    [user_country] => BR
    [merge_vars] => Array
        (
            [MEMBER_COUNT] => 4.2 million
            [FNAME] => Dee Thor Test 101
            [CAMPAIGN_TITLE] => Out-of-the-Closet Cards
            [CAMPAIGN_LINK] => https://thor.dosomething.org/us/campaigns/out-closet-cards?source=node/5740
            [CALL_TO_ACTION] => Make cards to help teachers create an LGBT-inclusive classroom.
            [STEP_ONE] => Distribute the Cards!
            [STEP_TWO] => Snap a Pic
            [STEP_THREE] => Find More Allies!
        )

    [email_template] => mb-campaign-signup-BR
    [subscribed] => 1
    [event_id] => 5740
    [email_tags] => Array
        (
            [0] => 5740
            [1] => drupal_campaign_signup
        )

    [activity_timestamp] => 1445463890
    [application_id] => US
)
```

**Comment by blisteringherb** [comment]
@DeeZone I think the underlying issue here is that for a BR user, their language is being saved as `en`. Do you know why that might be based on this particular test?

**Comment by deezone** [comment]
@blisteringherb Good point! I was working from the dev branch rather than the global branch when testing / creating issue branches. I suspect this issue will be closed out when I test again using the correct base branch.

**Comment by deezone** [comment]
@blisteringherb Updated sample payload, `user_language` and `user_country` bug has been fixed ( #5553 ). They're now in sync. This issue is about a `BR` / `pt-br` signing up for an English campaign. The template should be `[email_template] => mb-campaign-signup-US` based on the campaign being English.

**Related**: The `campaign_language` value should also be in the payload of both the `campaign_signup` and `campaign_reportback` transactions. The Message Broker system will use these values when generating the Global Digest.

**Related duplicate**:
- https://github.com/DoSomething/phoenix/issues/5568

**Comment by mikefantini** [comment]
Also is this still an issue? Is this just a duplicate of #5568? Can we close one of them if that is the case?
